src/pages/display/index.js

```diff
@@ -579,12 +579,11 @@ export default class Display extends Component {
       })
     }
 
-    this.updateLocation('collabs')
-  }
-
-  updateLocation = (slug) => {
-    const { subjkt, wallet } = this.state
-    this.props.history.push(`/${subjkt === '' ? wallet : subjkt}/${slug}`)
+    if (this.state.subjkt !== '') {
+      this.props.history.push(`/${this.state.subjkt}/collabs`)
```

Comment:
Can't we just always resolve to the address?

Reply:
hmm, it's done like this in the other tabs as well and ensures that the url contains the username.

Reply:
Ah my bad, I thought routing handled these